### ISSUE TITLE
Express missing from dependencies

### ISSUE THREAD

**Issue by aendrew** [searchkit/searchkit-boilerplate#8]
I'm trying to deploy this to Heroku and having a number of issues; most notably, the server won't come up because Express isn't listed as a dependency.

This is easily resolved by adding Express to package.json.

**Comment by Wozacosta** [comment]
Hey, did end manage to deploy it to Heroku?

**Comment by aendrew** [comment]
@Wozacosta Yep! The big thing was just adding Express and any other missing dependencies (there was another missing IIRC, forget what it was.). It's a private repo at the minute but I'll post a link here if we ever publish it.
